test/e2e/e2e_test.go

```diff
@@ -69,11 +74,8 @@ func TestMasterMachineCreated(t *testing.T) {
 	// validate virtual machine fields match the spec
 }
 
-func createTestClients() (*Clients, error) {
-	kubeconfig := os.Getenv("KUBE_CONFIG")
-	if kubeconfig == "" {
-		return nil, fmt.Errorf("KUBE_CONFIG environment variable is not set")
-	}
+func createTestClients(KubeConfig string) (*Clients, error) {
+	kubeconfig := os.Getenv(KubeConfig)
```

Comment:
```suggestion
	kubeconfig := os.Getenv(kubeConfig)
```

Reply:
I don't think so we would require os.Getenv(kubeConfig) again since the passed value to createTestClients would be having kubeConfig from TestMasterMachineCreated

Reply:
Also I got a bit confused in casing since for functions it's TestMasterMachineCreated and createTestClients, so perhaps the casing for the former function needs to be changed

Reply:
@Avni-Sharma You are correct. Let's remove that line. L78. We will just use the kubeConfig that has been passed down as the param. Editing my suggestion above

Reply:
The casing should remain as-is for the method names. We need Pascal case for public test methods and camelCase for private methods.

Reply:
Ack, Thanks